### PR TITLE
Some type fixes and mapping table improvements

### PR DESCRIPTION
**Pull request by kenchris**
@beaufortfrancois

***
<a href="https://pr-preview.s3.amazonaws.com/kenchris/web-nfc/pull/394.html" title="Last updated on Oct 18, 2019, 8:45 PM UTC (ca2920c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/web-nfc/394/0ab6912...kenchris:ca2920c.html" title="Last updated on Oct 18, 2019, 8:45 PM UTC (ca2920c)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4208,7 +4238,7 @@ <h3>Parsing content</h3>
       |record:NDEFRecord|, run these steps:
       <ol class=algorithm>
         <li>
-          Set |record|'s recordType to "`url`".
+          Set |record|'s recordType to "`absolute-url`".
```
Comment (kenchris) [inline index.html:4261]: This is a fix

File: index.html [hunk 2]
```diff
@@ -1873,74 +1880,97 @@ <h2>The <dfn>record type</dfn> string</h2>
...
-      <td><i>[= local type =]</i></td>
+      <td><i>[=local type name=]</i></td>
       <td><i>unused</i></td>
       <td>
         {{BufferSource}}
       </td>
-      <td><a>External type record</a> (TNF=4) with TYPE= [= local type =] inside
-        the <a>NDEFMessage</a> payload of another record</td>
+      <td>[=Local type=] record*</td>
+      <td>1</td>
```
Comment (kenchris) [inline index.html]: And this is a fix as well.

File: index.html [hunk 3]
```diff
@@ -1873,74 +1880,97 @@ <h2>The <dfn>record type</dfn> string</h2>
...
     <tr>
-      <td><i>[= local type =]</i></td>
+      <td><i>[=local type name=]</i></td>
       <td><i>unused</i></td>
       <td>
         {{BufferSource}}
       </td>
-      <td><a>External type record</a> (TNF=4) with TYPE= [= local type =] inside
-        the <a>NDEFMessage</a> payload of another record</td>
+      <td>[=Local type=] record*</td>
```
Comment (kenchris) [inline index.html]: We should add a section called "Local type record" - it is missing

Comment (beaufortfrancois) [reply index.html]: Since it seems to be ordered by TNF, shall we move "local type" row under "smart-poster" row?

File: index.html [hunk 4]
```diff
@@ -1983,8 +2013,8 @@ <h2>The <dfn>record type</dfn> string</h2>
       <td>""</td>
     </tr>
     <tr>
-      <td><a>External type record</a> (TNF=4) with TYPE=<a>external type</a></td>
-      <td><a>external type</a></td>
+      <td><a>External type record</a> (TNF=4) with TYPE=<a>external type name</a></td>
+      <td><a>external type name</a></td>
```
Comment (beaufortfrancois) [inline index.html]: Shall we add "local type name" as well below?

File: index.html (hunk 1 above)
Comment (beaufortfrancois) [reply index.html:4261]: Thx!

**Review by beaufortfrancois — APPROVED** [review]
LGTM with nits

**Comment by kenchris** [comment]
Fixed nits

File: index.html [hunk 5]
```diff
@@ -538,25 +538,28 @@ <h4>
       Well-known type records (TNF 1)
     </h4>
     <p>
-      The NFC Forum has standardized a small set of useful record types in
+      The NFC Forum has standardized a small set of useful sub record types in
```
Comment (zolkis) [inline index.html:541]: Why is "sub" needed?

**Pull request merged**